apps/avatax/bruno/Checkout/4_Create_checkout_US.bru

```diff
@@ -100,4 +100,5 @@ body:graphql:vars {
 
 vars:post-response {
   checkoutId: res.body.data.checkoutCreate.checkout.id
+  shippingMethodId: res.body.data.checkoutCreate.checkout.shippingMethods[1].id
```

Comment:
suggestion: I think we should use first shippingMethod, in case there's only 1

```suggestion
  shippingMethodId: res.body.data.checkoutCreate.checkout.shippingMethods[0].id
```